packages/kbn-content-management-utils/src/saved_object_content_storage.ts

```diff
@@ -69,11 +69,13 @@ function savedObjectToItem<Attributes extends object>(
     error,
     namespaces,
     version,
+    managed,
```

Comment:
By default, `managed` is false. The only way to change to true is to use SO import, with an [option with conditions](https://github.com/elastic/kibana/issues/140364)